src/package/package_api.js

```diff
@@ -84,7 +84,7 @@ function defineReadOnlyProperty(object, key, value) {
 function PackageInformation(json) {
   for (var field in json) {
     var val = json[field];
-    if (field === 'installDate')
+    if (field === 'lastModified')
```

Comment:
Thanks for grab this mistake. It is definitely lgtm. BTW, the official tizen api reference is [2.3.0] (https://developer.tizen.org/documentation/dev-guide/2.3.0)